components/query_handler/queryHandler.py

```
@@ -1,35 +1,48 @@
-            await websocket.send(json.dumps(message))
-
-        else:
-            message["response"] = f"Unknown message type: {message['code']}"
-            message["error"] = True
-            await websocket.send(json.dumps(message))
-
+        try:
+            message = json.loads(message)   
+            response = handleQuery(message=message)   
+            await websocket.send(json.dumps(response))  
+        except json.JSONDecodeError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)